frontend/components/Select.vue

```diff
@@ -27,18 +27,20 @@
         aria-label="Default select example"
       >
         <option
+          class="text-xs md:text-sm"
           selected
           style="font-family: 'Inter', sans-serif"
-          class="text-black font-medium"
         >
-          Job Title
+          Job title
         </option>
         <option
           style="font-family: 'Inter', sans-serif"
-          class="text-blackfont-medium"
+          v-for="(link, index) in jobtitles"
+          :key="index"
           value="1"
```

Comment:
Hi @Asam237  nice job.
but since you are make a select why are value fixed to 1 for all option. i think it will be i ggod idea to habe one value for each option. something like :value="link.id" could be better so that we ca use it later to make some request.

Reply:
Because at the BackEnd response level, I just have a character table... Should we add this field at the BackEnd level @elhmn ?
![jta](https://user-images.githubusercontent.com/34966088/153135586-58d0d4a1-f7cf-4695-8862-dfc5d11d3f42.png)

Reply:
oki i see but in this case you can either asign the string to value or assign the index to the value. i think the job title in this case will be better, so :value="link"